Azure/sonic-buildimage
# [Pmon] Add new daemon "syseepromd" to pmon docker

**- What I did**
Introduce a new daemon syseepromd to pmon, which will post syseeprom info to pmon when starting pmon, and will clear those info when stopping pmon, and will keep monitoring the DB in case the info was cleared.


**this PR has a dependency on https://github.com/Azure/sonic-platform-daemons/pull/25**

**- How I did it**
1. modify make file to add the new daemon syseepromd
2. add code to generate supervisord conf and start.sh for the new daemon 


**- How to verify it**
build an image include this PR to verify the state DB,  thes syseeprom info should be there after pmon started, stop pmon and those info should be removed from the state DB.

**- Description for the changelog**
```
dockers/docker-platform-monitor/start.sh.j2
/dockers/docker-platform-monitor/docker-pmon.supervisord.conf.j2
rules/docker-platform-monitor.mk
rules/sonic-post-syseeprom.mk
```


**- A picture of a cute animal (not mandatory but encouraged)**
